README.rst

```diff
@@ -1,7 +1,7 @@
 `BinderHub`_
 ============
 
-.. image:: https://readthedocs.org/projects/binderhub/badge/?version=latest
+.. image:: http://readthedocs.org/projects/binderhub/badge/?version=latest
```

Comment:
It can. This is a leftover of trying to work around an error in the latest Sphinx that warns for non-local images. I just cut and pasted the RTD sample code. Going to change it back to https. Good catch.